.eslintignore

```diff
@@ -1 +1,8 @@
 src/**/__tests__/*
+public/generated/*
+/node_modules/*
+/coverage/*
+/.swc/*
+/.next/*
+/out/*
+/storybook-static/*
```

Comment:
A lot of the files in these directories are html or json, but it was still scanning through them because of our very permissive globs in tsconfig.json (eslint reads from that)

```
  "include": [
    "next-env.d.ts",
    "additional.d.ts",
    "src/**/*",
    "**/*.ts",
    "**/*.tsx",
    "**/*.js",
    "**/*.jsx"
  ],
```